scripts/createERC20Rollup.ts

```diff
@@ -0,0 +1,32 @@
+import { ethers } from 'hardhat'
+import '@nomiclabs/hardhat-ethers'
+import { Signer } from 'ethers'
+import { ERC20PresetFixedSupply__factory } from '../build/types/factories/@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply__factory'
+import { createRollup } from './rollupCreation'
+
+async function deployERC20Token(deployer: Signer): Promise<string> {
+  const factory = await new ERC20PresetFixedSupply__factory(deployer).deploy(
+    'FeeToken',
+    'FEE',
+    ethers.utils.parseEther('1000000000'),
+    await deployer.getAddress()
+  )
+  const feeToken = await factory.deployed()
+
+  return feeToken.address
+}
+
+async function main() {
+  const [deployer] = await ethers.getSigners()
+  const customFeeTokenAddress = await deployERC20Token(deployer)
```

Comment:
I think this should include the option to specify an existing token

Reply:
done